### PR TITLE
add collator options

### PR DESCRIPTION
**Pull request by romulocintra**
Update collation options.

https://github.com/tc39/ecma402/pull/459

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/intl/collator/collator/index.md [hunk 1]
```diff
@@ -46,13 +46,11 @@ new Intl.Collator(locales, options)
...
     - `co`
       - : Variant collations for certain locales. Possible values include:
-        "`big5han`", "`dict`", "`direct`",
-        "`ducet`", "`gb2312`", "`phonebk`",
-        "`phonetic`", "`pinyin`", "`reformed`",
-        "`searchjl`", "`stroke`", "`trad`",
-        "`unihan`". This option can be also be set through the
-        `options` property
-        "`collation`".
+        "`big5han`", "`compat`", "`dict`", "`direct`",
```
Comment (wbamberg) [inline files/en-us/web/javascript/reference/global_objects/intl/collator/collator/index.md:49]: This would be a lot easier to read as a list.

**Review by wbamberg — COMMENTED** [review]
I think we should consider formatting this as list.

Also, forgive my ignorance here, but following the links, it looks like these values are coming ultimately from https://github.com/unicode-org/cldr/blob/latest/common/bcp47/collation.xml#L13-L31, but there are some values listed there that are not included here (for example, `emoji`, `standard`, `search`). Is this intended?

File: files/en-us/web/javascript/reference/global_objects/intl/collator/collator/index.md (hunk 1 above)
Comment (romulocintra) [reply files/en-us/web/javascript/reference/global_objects/intl/collator/collator/index.md]: Already updated to be a list :)

**Comment by romulocintra** [comment]
> I think we should consider formatting this as list.
> 
> Also, forgive my ignorance here, but following the links, it looks like these values are coming ultimately from https://github.com/unicode-org/cldr/blob/latest/common/bcp47/collation.xml#L13-L31, but there are some values listed there that are not included here (for example, `emoji`, `standard`, `search`). Is this intended?

Hi @wbamberg  search and standard don't have support on ECMA402 https://tc39.es/ecma402/#sec-intl-collator-internal-slots , emoji still unclear the support, meaning as soon as we have it sure we will include.

File: files/en-us/web/javascript/reference/global_objects/intl/collator/collator/index.md [hunk 2]
```diff
@@ -46,13 +46,23 @@ new Intl.Collator(locales, options)
...
-        "`ducet`", "`gb2312`", "`phonebk`",
-        "`phonetic`", "`pinyin`", "`reformed`",
-        "`searchjl`", "`stroke`", "`trad`",
-        "`unihan`". This option can be also be set through the
-        `options` property
-        "`collation`".
+          - `big5han`
+          - `dict`
+          - `direct`
+          - `compat`
```
Comment (wbamberg) [inline files/en-us/web/javascript/reference/global_objects/intl/collator/collator/index.md:52]:
```suggestion
          - `compat`
          - `dict`
          - `direct`
```

File: files/en-us/web/javascript/reference/global_objects/intl/collator/collator/index.md [hunk 3]
```diff
@@ -126,14 +136,23 @@ new Intl.Collator(locales, options)
...
-        "`big5han`", "`dict`", "`direct`",
-        "`ducet`", "`gb2312`", "`phonebk`" (only
-        supported in German), "`phonetic`", "`pinyin`",
-        "`reformed`", "`searchjl`", "`stroke`",
-        "`trad`", "`unihan`".
-
+          - `big5han`
+          - `dict`
+          - `direct`
+          - `compat`
```
Comment (wbamberg) [inline files/en-us/web/javascript/reference/global_objects/intl/collator/collator/index.md:143]:
```suggestion
          - `compat`
          - `dict`
          - `direct`
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks @romulocintra for the clarification and formatting the options as lists. Just a tweak to list them in alphabetical order.

**Review by wbamberg — APPROVED** [review]
👍 thank you @romulocintra !

**Pull request merged**